conf.yaml

```diff
@@ -791,7 +791,7 @@ contents:
             prefix:     en/cloud-on-k8s
             tags:       Kubernetes/Reference
             subject:    ECK
-            current:    1.2
+            current:    1.3
             branches:   [ master, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
```

Comment:
We should also add 1.3 into the `branches` list as well. We can do that as soon as the branch is ready to build (even before we want to make it `current`), but we can also do both in the same PR as well.

Reply:
Looks like this error got caught by CI 🙌 : 
```
15:47:34 INFO:build_docs:Current branch <1.3> is not in <branches> in book <Elastic Cloud on Kubernetes> at lib/ES/Book.pm line 118.
```

Reply:
Sorry, just saw #1996. 🤦

Reply:
No worries @gtback, thanks for being proactive. I'm planning to merge #1996 and re-kick CI here on Monday.